test/integration/controllermanager/seed/extensionscheck/extensionscheck_suite_test.go

```diff
@@ -89,11 +98,26 @@ var _ = BeforeSuite(func() {
 	mgr, err := manager.New(restConfig, manager.Options{
 		Scheme:             kubernetes.GardenScheme,
 		MetricsBindAddress: "0",
+		NewCache: cache.BuilderWithOptions(cache.Options{
+			SelectorsByObject: map[client.Object]cache.ObjectSelector{
+				&gardencorev1beta1.ControllerInstallation{}: {
+					Label: labels.SelectorFromSet(labels.Set{testID: testRunID}),
+				},
+				&gardencorev1beta1.Seed{}: {
+					Label: labels.SelectorFromSet(labels.Set{testID: testRunID}),
+				},
+			},
+		}),
 	})
 	Expect(err).NotTo(HaveOccurred())
 
+	fakeClock = testclock.NewFakeClock(time.Now())
+	//This is required so that the ExtensionsReady condition is created with appropriate lastUpdateTimestamp and lastTransitionTimestamp.
```

Comment:
```suggestion
	// This is required so that the ExtensionsReady condition is created with appropriate lastUpdateTimestamp and
	// lastTransitionTimestamp.
```